lib/fog/aws/requests/compute/create_vpc.rb

```diff
@@ -11,7 +11,6 @@ class Real
         # * options<~Hash>:
         #   * InstanceTenancy<~String> - The allowed tenancy of instances launched into the VPC. A value of default
         #     means instances can be launched with any tenancy; a value of dedicated means instances must be launched with tenancy as dedicated.
-        #     please not that the documentation is incorrect instanceTenancy will not work while InstanceTenancy will
```

Comment:
Is this no longer true? Do we need to worry about backwards compatibility issues?

Reply:
It's true that lowercase won't work, but it's not true that the documentation is incorrect anymore. That link to documentation shows Capital I too, hence no problem with backward compatibility either. It was just a documentation update on their end.

Reply:
Got it, thanks for clarifying. Sounds good.